compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaSymbol.java

```diff
@@ -65,7 +66,16 @@ protected BallerinaSymbol(String name, SymbolKind symbolKind, BSymbol symbol, Co
 
     @Override
     public Optional<String> getName() {
-        return Optional.ofNullable(this.name);
+        // In the langlib context, reserved keywords can be used as regular identifiers. Therefore, they will not be
+        // escaped.
+        if (getModule().isPresent() && getModule().get().id() != null &&
+                !getModule().get().id().moduleName().isEmpty() &&
+                getModule().get().id().moduleName().startsWith("lang.") &&
+                !getModule().get().id().orgName().isEmpty() &&
+                getModule().get().id().orgName().startsWith("ballerina")) {
```

Comment:
Shall we refactor this a bit? Assign the returned value to a var and use, so that all these `getModule().get().id()` calls are simplified?

Reply:
Updated.